main.tex

```diff
@@ -774,22 +790,21 @@ \subsubsection{Support for table concatenation}
 tables have the same columns, we may want to stack them ``vertically'' to create a
 new table with the same columns but all rows. If two tables are row-matched but
 have distinct columns, we may want to stack them ``horizontally'' to create a
-new table with the same rows but all columns. For other situations, more general
-table concatenations or joins are also possible when two tables share some
-common columns.
+new table with the same rows but all columns. For other situations, more generic
+table concatenation or joining are also possible when two tables share some
+columns.
 
 \subsubsection{Using astropy objects in tables}
 
-The \texttt{Table} object now allows \texttt{Quantity} objects, celestial
-coordinate objects (\texttt{SkyCoord}), and date/time objects (\texttt{Time}) to
-be used as columns, and also provides a general way for other user-defined
-objects to be used as columns. This makes it possible for example to easily
-represent catalogs of sources or time series as in Astropy, and having both the
-benefits of the \texttt{Table} object (such as accessing specific rows/columns
-or groups of rows/columns, combining tables, and so on) and the benefits of e.g.
-the \texttt{SkyCoord} or \texttt{Time} classes (such as converting the
-coordinates to a different frame, or accessing the date/time as a modified
-Julian date or in another time scale).
```

Comment:
Note: I am unsure why MJD was singled out. Here, I attempted to reduce the complexity of the paragraph to make it easier to read. I can revert if any of my changes are undesirable.